cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -64,6 +68,19 @@ func PerformPostUpgradeTasks(client clientset.Interface, cfg *kubeadmapi.MasterC
 		errs = append(errs, err)
 	}
 
+	// Don't fail the upgrade phase if failing to determine to backup kube-apiserver cert and key.
+	if shouldBackup, err := shouldBackupAPIServerCertAndKey(newK8sVer); err != nil {
```

Comment:
for readability, please move `shouldBackup, err := shouldBackupAPIServerCertAndKey(newK8sVer)` out of this if clause